docs_src/microservices/application/Triggers.md

```diff
@@ -1,109 +1,198 @@
 ```
 
+!!! edgey "Edgex 2.0"
+    For Edgex 2.0 Redis is now the default EdgeX MessageBus implementation used. Also, the Redis implementation changed from `Redis streams` to `Redis Pub/Sub`, thus the type value changed from `redisstreams` to `redis`
+
 !!! important
-    When using ZMQ for the message bus, the Publish Host **MUST** be different for every topic you wish to publish to since the SDK will bind to the specific port. 5563 for example cannot be used to publish since `EdgeX Core Data` has bound to that port. Similarly, you cannot have two separate instances of the app functions SDK running publishing to the same port.
+    When using ZMQ for the message bus, the Publish Host **MUST** be different for each publisher to since the they will bind to the specific port. 5563 for example cannot be used to publish since `EdgeX Core Data` has bound to that port. Similarly, you cannot have two separate instances of the app functions SDK running and publishing to the same port. This is why once Device services started publishing the the EdgeX MessageBus the default was changed to `Redis Pub/Sub`
 
 !!! note
     When using MQTT for the message bus, there is additional configuration required for specifying the MQTT specific options. 
 
-Here is example `MessageBus` configuration when using MQTT as the message bus:
+### Example using MQTT
+
+Here is example `EdgexMessageBus` configuration when using MQTT as the message bus:
 
 ```toml
-[MessageBus]
+[Trigger.EdgexMessageBus]
 Type = "mqtt"
-    [MessageBus.SubscribeHost]
-        Host = "localhost"
-        Port = 1883
-        Protocol = "tcp"
-    [MessageBus.PublishHost]
-        Host = "localhost"
-        Port = 1883
-        Protocol = "tcp"
-    [MessageBus.Optional]
-        # MQTT Specific options
-        Username =""
-        Password =""
-        ClientId ="AppService"
-        # Connection information
-        Qos          =  "0" # Quality of Sevice 0 (At most once), 1 (At least once) or 2 (Exactly once)
-        KeepAlive    =  "10" # Seconds (must be 2 or greater)
-        Retained     = "false"
-        AutoReconnect  = "true"
-        ConnectTimeout = "5" # Seconds
-        SkipCertVerify = "false"
-        CertFile       = ""
-        KeyFile        = ""
-        KeyPEMBlock    = ""
-        CertPEMBlock   = ""
+    [Trigger.EdgexMessageBus.SubscribeHost]
+    Host = "localhost"
+    Port = 1883
+    Protocol = "tcp"
+    SubscribeTopics="edgex/events/#"
+    [Trigger.EdgexMessageBus.PublishHost]
+    Host = "localhost"
+    Port = 1883
+    Protocol = "tcp"        
+    PublishTopic="" # optional if publishing response back to the MessageBus
+    [Trigger.EdgexMessageBus.Optional]
+    # MQTT Specific options
+    ClientId ="new-app-service"
+    Qos            = "0" # Quality of Sevice values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
```

Comment:
```suggestion
    Qos            = "0" # Quality of Service values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
```

Reply:
Sevice ==> Service

Reply:
Try this commit suggestion thing again...